septentrion/core.py

```diff
@@ -2,11 +2,28 @@
 This is where the migration plan is computed, by merging information
 from the existing files (septentrion.files) and from the db (septentrion.db)
 """
-
+import logging
 from typing import Any, Dict, Iterable, Optional
 
 from septentrion import configuration, db, exceptions, files, style, utils, versions
 
+logger = logging.getLogger(__name__)
+
+
+def initialize(**kwargs):
+    settings = configuration.Settings(**kwargs)
+
+    level = configuration.log_level(verbosity=settings.VERBOSITY)
+    logging.basicConfig(level=level)
+    logger.info("Verbosity level: %s", logging.getLevelName(level))
```

Comment:
I'm all for create_table in core, but a library should not mess with logs. I think that part belongs in cli.

Reply:
ah yes you're right